DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ReviewConfirmationRequests.cshtml

```diff
@@ -77,7 +77,7 @@
                         </td>
                         <td role="cell" class="nhsuk-table__cell nhsuk-u-font-size-16">
                             <span class="nhsuk-table-responsive__heading">Request </span>
-                            @competency.SupervisorName<br />
+                            @competency.SupervisorName (@question.CentreName)<br />

```

Comment:
Why is SupervisorName attached to @competency whereas CentreName is attached to @question object? Feels like they should be properties of the same object?

Reply:
Corrected.